code/UI/General/TabMenus/Scoreboard/Entry/ScoreboardEntry.cs

```diff
@@ -45,4 +77,28 @@ public void Update()
 
 		PlayerAvatar.SetTexture( $"avatar:{_client.PlayerId}" );
 	}
+
+	public void OnClick()
+	{
+		// Only allow tagging currently alive players, and only if the selected player is not the local player
+		if ((_client.Pawn as Player).Status is not PlayerStatus.Alive || _client.Pawn.IsLocalPawn)
```

Comment:
You can remove the comment, I think this explains it enough. 
```suggestion
		if ( !_client.Pawn.IsAlive() || _client.Pawn.IsLocalPawn )
```